includes/admin.form.inc

```diff
@@ -360,20 +360,42 @@ function islandora_scholar_admin_pdf_form($form, &$form_state) {
  *   Drupal form definition.
  */
 function islandora_scholar_admin_xpath_form($form, &$form_state) {
+  form_load_include($form_state, 'inc', 'islandora_scholar', 'includes/admin.form');
+  if (!empty($form_state['values']['islandora_scholar_xpaths_demo'])) {
+    $islandora_scholar_xpath_pid = $form_state['values']['islandora_scholar_xpaths_demo'];
+    exec($islandora_scholar_xpath_pid, $output, $return_value);
+    $islandora_scholar_xpaths_general_description = t('These XPaths are shared within most of the submodules. To see the MODS value after editing an entry, tab to the next field. Testing against pid: !pid <img src="', array(
+        '!pid' => "$islandora_scholar_xpath_pid",
+      )) . url('misc/watchdog-ok.png') . '"/><br/><br/>';
```

Comment:
I think it would be better to write "complete" sentences in `t()`, as in
```php
      $islandora_scholar_xpaths_general_description = t('These XPaths are shared within most of the submodules. To see the MODS value after editing an entry, tab to the next field. Testing against pid: !pid <img src="!img_url"/><br/><br/>', array(
        '!pid' => "$islandora_scholar_xpath_pid",
        '!img_url' => url('misc/watchdog-ok.png'),
      ));
```
to be easier on translators... Similarly for [L370](https://github.com/Islandora/islandora_scholar/pull/299/commits/e5216a82ff3b166d0c8c98db21689fa39c7fa525#diff-0cdd19abe14e51fe9e7f061ec72c86d3R370), [L373](https://github.com/Islandora/islandora_scholar/pull/299/commits/e5216a82ff3b166d0c8c98db21689fa39c7fa525#diff-0cdd19abe14e51fe9e7f061ec72c86d3R373) and [L376](https://github.com/Islandora/islandora_scholar/pull/299/commits/e5216a82ff3b166d0c8c98db21689fa39c7fa525#diff-0cdd19abe14e51fe9e7f061ec72c86d3R376).

Also, is it safe to use `!placeholder` (especially `!pid`, since that is unprocessed user input)?